api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -277,5 +277,5 @@ message GrpcJsonTranscoder {
   // body was oversized.
   //
   // If unset or zero, the current stream buffer size is used.
-  uint32 max_message_size = 15;
+  uint32 max_body_size = 15;
```

Comment:
We need to split it into two fields:  `max_request_body_size` and max_response_body_size`

Reply:
Done.